.github/workflows/e2e.yml

```diff
@@ -7,68 +7,106 @@ on:
+      be_url: ${{ inputs.be_url }}
       project: ${{ inputs.project }}
-      browser: 'chrome'
-      test_id: 'fixture'
-      test_type: ':e2e:'
-      runner: 'ubuntu-latest'
-  e2e_fe_chrome:
-    needs:
-      - e2e_be
-      - e2e_fe_smoke
+      test_id: "fixture"
+      parallel: 10
+      tag: 'sdp-dev-r1-chrome'
+  fe_core_safari:
+    needs: fe_smoke
     uses: dpc-sdp/github-actions/.github/workflows/run_e2e_fe.yml@main
     secrets: inherit
     with:
-      tags: "(@core or @${{ inputs.project }}) and @regression and not @mobile"
-      be_url: ${{ inputs.be_url }}
+      name: 'FE Core Safari'
+      test_type: ':e2e:'
+      tags: '@core and @regression and not @mobile'
       fe_url: ${{ inputs.fe_url }}
+      be_url: ${{ inputs.be_url }}
       project: ${{ inputs.project }}
-      browser: 'chrome'
-      test_id: 'fixture'
+      test_id: "fixture"
+      parallel: 1
+      tag: 'sdp-dev-r1-chrome'
+      browser: 'browserstack.safari'
+  fe_core_android_chrome:
+    needs: fe_smoke
+    uses: dpc-sdp/github-actions/.github/workflows/run_e2e_fe.yml@main
+    secrets: inherit
+    with:
+      name: 'FE Core Android Chrome'
       test_type: ':e2e:'
-      runner: 'ubuntu-latest'
-  e2e_fe_android:
-    needs:
-      - e2e_be
-      - e2e_fe_smoke
+      tags: '@core and @regression and not @desktop'
+      fe_url: ${{ inputs.fe_url }}
+      be_url: ${{ inputs.be_url }}
+      project: ${{ inputs.project }}
+      test_id: "fixture"
+      parallel: 1
+      tag: 'sdp-dev-r1-chrome'
+      browser: 'browserstack.androidChrome'
+  fe_core_iphone_safari:
+    needs: fe_smoke
     uses: dpc-sdp/github-actions/.github/workflows/run_e2e_fe.yml@main
     secrets: inherit
     with:
-      tags: "(@core or @${{ inputs.project }}) and @regression and not @desktop"
+      name: 'FE Core iPhone Safari'
+      test_type: ':e2e:'
+      tags: '@core and @regression and not @desktop'
+      fe_url: ${{ inputs.fe_url }}
       be_url: ${{ inputs.be_url }}
+      project: ${{ inputs.project }}
+      test_id: "fixture"
+      parallel: 1
+      tag: 'sdp-dev-r1-chrome'
+      browser: 'browserstack.iphoneSafari'
+  fe_core_visual:
+    needs: fe_smoke
+    uses: dpc-sdp/github-actions/.github/workflows/run_e2e_fe.yml@main
+    secrets: inherit
+    with:
+      name: 'FE core visual'
+      app: frontend-visual
+      test_type: ":applitools:"
+      eyes_appname: 'sdp-core-v1'
+      tags: "@core and @visual-regression-minimal"
       fe_url: ${{ inputs.fe_url }}
+      be_url: ${{ inputs.be_url }}
       project: ${{ inputs.project }}
-      browser: 'androidChrome'
-      test_id: 'fixture'
-      test_type: ':e2e:'
-      runner: 'ubuntu-latest'
+      test_id: "fixture"
+      parallel: 1
+      tag: 'sdp-dev-r1-chrome'
   set_status:
-    needs: [e2e_be, e2e_fe_smoke, e2e_fe_chrome, e2e_fe_android]
+    needs: [fe_core, fe_core_safari, fe_core_iphone_safari, fe_core_android_chrome, fe_core_visual]
```

Comment:
Do we need to add `fe_smoke`?
What happens if `fe_smoke` job fails? Does this `set_status` job still execute?

Reply:
> Do we need to add `fe_smoke`? What happens if `fe_smoke` job fails? Does this `set_status` job still execute?

Yes, good question, will add it. Thanks!